news/2017-11-28-beatmap-spotlights-october-2017.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "Beatmap Spotlights October 2017"
+date: 2017-11-28 16:00:00 +0000
+---
+
+Halloween may be over, but with that comes a selection of the best maps from the spooky month of October. It’s time to look at this month’s Spotlights, and maybe get a medal while you’re at it!
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+The beatmap charts for the October Spotlights can be found [here](https://osu.ppy.sh/p/chart?ch=MONTH1711). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access.
+
+## Past Spotlights results
+
+Congratulations to the winners of the September 2017 Spotlights. They receive one month of supporter status for their efforts!
```

Comment:
change `receive` to `received` if past tense
`They receive one month` -> `They received one month`

or add `will` between `They` and `receive` if it is still pending to happen?
`They receive one month` -> `They will receive one month`

Reply:
"will receive" is best